public/_redirects

```diff
@@ -0,0 +1 @@
+/* /index.html 200
```

Comment:
is that for netlify?

Reply:
Yes. Since Cambiatus is an SPA, we need to make netlify redirect everything to index and let the application handle routing

Reply:
oh, then Netlify is not using our docker image? Because it already does that for us. We solve this problem using nginx

Reply:
No, it runs `yarn build` and hosts the output directory. I'll look into using our docker image, I'm not sure if Netlify supports it

Reply:
Nice! If we can make it work with docker we will have the same env in prod and in Netlify